package.json

```diff
@@ -1,33 +1,47 @@
 {
   "name": "shield-studies-addon-utils",
-  "description": "Utilities for building Shield-Study Mozilla Firefox addons.",
-  "version": "4.1.0",
-  "author": "Gregg Lind <glind@mozilla.com>",
+  "description": "Utilities for building Shield-Study Mozilla Firefox add-ons.",
+  "version": "5.0.0-alpha1",
```

Comment:
Update to reflect RC / release status: 5.0.0-rc1 or 5.0.0 depending on at what stability-level we merge this PR

Reply:
Thanks!  This missed this!